### ISSUE TITLE
Single Onward Journey - Skip link and title element

### ISSUE THREAD

**Issue by pjlee11** [bbc/simorgh#685]
**Is your feature request related to a problem? Please describe.**
The SOJ component needs an visually hidden skip link which will read "Skip ${title}". EG: "Skip More on Archaeology", and a visual title 'More on Archaeology'.

**Describe the solution you'd like**
Create an visually hidden `<a>` which will act as a skip link for screen reader/keyboard users. 
- Should be visually displayed on focus
- Needs to anchor to the last element in the component, see https://github.com/BBC-News/simorgh/issues/689
- The href used for the anchor should be user friendly and descriptive (as it is user facing), the following would be ideal: `href="#end-of-more-on-archaeology"`

Create a `<strong>` element which will act as the visual title for the single onward journey component with the following CSS and attributes:
- Great primer typography specs
- colour: `#404040` (C_STORM)
- padding 16 top and bottom
- padding left 12px (due to the 4px border on the main component this is 12px to make the overall spacing 16px)
- aria hidden true

**Additional context**

**Screenshots**
![image](https://user-images.githubusercontent.com/7791726/45832993-5db8b780-bcfb-11e8-92ec-d6a40aee2259.png)
![image](https://user-images.githubusercontent.com/7791726/45831254-cfdacd80-bcf6-11e8-88f1-5ef47e75cfe4.png)

**Comment by pjlee11** [comment]
Additional screenshots left out due to UX needing to update the design documents

**Comment by pjlee11** [comment]
@greenc05 could you confirm whether the "More from Archaeology" title should be a `<span>` or a `<h2>`. Considering it is going to be hidden from screen readers makes me think it could be a `<span>`, however it may want to be included in the heading order so would want to be a `<h2>`. This could also be mitigated by adding a landmark.

Could you clarify how this should be implemented for accessibility best practice, thanks!

**Comment by greenc05** [comment]
@pjlee11, I had actually thought that should be a strong tag, to give it some semantic meaning. A heading isn't appropriate as all the content that is below this component would also sit under this heading. The mark up (along with landmark info) in the screen reader UX doc should be followed unless you have any better solutions. I do have an implementation example which I will send you.

**Comment by greenc05** [comment]
@pjlee11 just re-reading your question, are you referring to the 'Skip more on Archaeology' link or the 2nd element in the component 'More on Archaeology'?  My answer above is for the later (the 2nd element)
